src/meta/processors/AuthenticationProcessor.cpp

```diff
@@ -0,0 +1,310 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
```

Comment:
please update to `2019`

Reply:
Sorry, my bad, let me check all new files.

Reply:
I have checked the license head, and all the files are 2018 currently. Do we need to be changed them to 2019?

Reply:
Changed to 2019 with all new files as below :
src/meta/processors/AuthenticationProcessor.cpp
src/meta/processors/AuthenticationProcessor.h
src/meta/test/AuthProcessorTest.cpp